src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--position calendar--month-length-28 calendar--month-length-29 calendar--month-length-30 calendar--month-length-31 calendar-start-day--wed">
```

Comment:
Remove multiple month-length modifiers. You should use only one modifier based on the month length.